### ISSUE TITLE
FileMetadataIT.testJsonParserWithDirectoryLabels failing

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#6863]
FileMetadataIT.testJsonParserWithDirectoryLabels is failing for me locally as well as at https://jenkins.dataverse.org/job/IQSS-dataverse-develop/435/

**Comment by scolapasta** [comment]
This test still is failing (at leas intermittently) so I'm reopening up this issue.

**Comment by scolapasta** [comment]
(Oh, I I didn't realize the April date above was 2020, not 2021 (i.e. a year ago). Still not much noise here, so I think it's ok to track here.

**Comment by sekmiller** [comment]
This is still failing via Jenkins in the Download All branch, though not for me locally in that branch.

**Comment by qqmyers** [comment]
Can using the first 8 chars of a UUID as the testName (line 44) cause issues? I.e. some char that isn't allowed in a dataverse name/alias?

**Comment by pdurbin** [comment]
@qqmyers yes, good call. It's probably the dv alias. If we switch to `UtilIT.getRandomDvAlias()` a "dv" will be prepended, making a valid dv alias (rather than all digits, occasionally).
